bots/notify/src/test/java/org/openjdk/skara/bots/notify/issue/IssueNotifierTests.java

```diff
@@ -47,6 +47,7 @@
 import static org.openjdk.skara.issuetracker.jira.JiraProject.SUBCOMPONENT;
 
 public class IssueNotifierTests {
+    private static final String JEP_NUMBER = "customfield_10701";
```

Comment:
You should be able to import this from JiraProject instead of duplicating the definition.